modules.d/95nvmf/parse-nvmf-boot-connections.sh

```diff
@@ -297,6 +297,22 @@ parse_nvmf_discover() {
     return 0
 }
 
+nvmf_timeout_udev_rule() {
+    local timeout
```

Comment:
For my edification, does this need to be posix compliant?  I guess `local` it is used elsewhere, but I thought this was a bash-ism really.

Reply:
Right but `local` is used a lot in the dracut code, also in scripts that pretend to be POSIX (`#!/bin/sh`).

According to [this stackoverflow article](https://stackoverflow.com/questions/18597697/posix-compliant-way-to-scope-variables-to-a-function-in-a-shell-script), all practically relevant shells support it, so it should be ok. the dracut checkers `shellcheck` and `shfmt` also don't take offense from it.